modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/DestinationType.java

```diff
@@ -1,10 +1,24 @@
 package org.wso2.andes.kernel;
 
+import org.wso2.andes.amqp.AMQPUtils;
```

Comment:
I would suggest otherwise. This would lead to cyclic dependency in the package which we want to resolve when doing C5 and we should make way to that in all the code we write.